main/template/default/admin/teacher_time_report.tpl

```diff
@@ -0,0 +1,129 @@
+{% extends "default/layout/main.tpl" %}
+
+{% block body %}
+    <script>
+        $(document).on('ready', function () {
+            $('#course').on('change', function () {
+                $('#session').prop('selectedIndex', 0);
+                $('#teacher').prop('selectedIndex', 0);
+            });
+
+            $('#session').on('change', function () {
+                $('#course').prop('selectedIndex', 0);
+                $('#teacher').prop('selectedIndex', 0);
+            });
+
+            $('#teacher').on('change', function () {
+                $('#course').prop('selectedIndex', 0);
+                $('#session').prop('selectedIndex', 0);
+            });
+
+            $('#daterange').daterangepicker({
+                format: 'YYYY-MM-DD',
+                startDate: new Date('{{ filterStartDate }}'),
+                endDate: new Date('{{ filterEndDate }}'),
+                maxDate: new Date('{{ filterMaxDate }}'),
+                separator: ' / ',
+                locale: {
+                    applyLabel: '{{ 'Ok' | get_lang }}',
+                    cancelLabel: '{{ 'Cancel' | get_lang }}',
+                    fromLabel: '{{ 'From' | get_lang }}',
+                    toLabel: '{{ 'Until' | get_lang }}',
```

Comment:
Fails because in French it is translated to "jusqu'au". To avoid it, all calls to get_lang should always be surrounded by *double* quotes, which are already escaped by get_lang: toLabel: "{{ 'Until' | get_lang }}'